src/lava/magma/core/run_configs.py

```diff
@@ -70,19 +70,21 @@ def select(self,
 class Loihi1SimCfg(RunConfig):
     """Run configuration selects appropriate ProcessModel -- either
     `SubProcessModel` for a hierarchical Process or else a `PyProcessModel`
-    for a standard Process.
+    for a standard Process. Implements and overrides `select()` method of the
+    parent class `RunConfig` to achieve this.
+
+    First, exceptions are checked in `exception_proc_model_map` dictionary
```

Comment:
First explain that this RunCfg has a default behavior of first returning SubProcModel and then LeafProcModels. However this default behavior can be overridden for specific Process classes...

Reply:
Done.